.github/workflows/build-and-test-windows.yml

```diff
@@ -28,7 +31,7 @@ jobs:
           - internal
           - other
     runs-on: windows-latest
-    if: ${{ github.actor != 'dependabot[bot]' }}
+    if: ${{ github.actor != 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'Run Windows') }}
```

Comment:
Will this block execution on when the PR is merged into main if the PR did not have the `Run Windows` label?

Reply:
Good point, need to fix that